.gitignore

```diff
@@ -9,6 +9,7 @@ platform/mac/build
 # Windows related files
 *Thumbs.db
 CMakeLists.txt.user
+external_libraries/asio*
```

Comment:
Thumbs.db is evident, right...
CMakeLists.txt.user is under Qt-Creator control, you wouldn't edit it.
asio* refers to the Steinberg ASIO-SDK - we use it in vanilla state, nobody would ever edit it (possibly even illegal), but it has to be in the source tree to be detected by portaudio.

I don't like "untracked files" that will never be added in my git status, and there is always the risk that somebody adds and commits them by accident (it has happened ;) )